web/themes/light/styles.css

```diff
@@ -1,113 +1,157 @@
-  margin-top: 0;
+    font-weight: 300;
+    font-size: 1.75rem;
+    line-height: 2.75rem;
+    letter-spacing: 0.013125rem;
+    color: #33475b;
+    padding: 1.125rem 0 0.875rem 0 !important;
+    margin: 0 calc(-1 * (2.5rem - 1.5rem)) 0 calc(-1 * (2.5rem - 1.5rem)) !important;
 }
 
-.theme-panel {
-  background-color: #fff;
-  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
-  padding: 30px;
+.theme-sub-heading {
+    margin-bottom: 1rem;
 }
 
-.theme-btn-provider {
-  background-color: #fff;
-  color: #333;
-  min-width: 250px;
+.theme-sub-heading p {
+    margin: 0;
+    padding: 0;
+}
+
+.theme-panel {
+    border-bottom-left-radius: 0.25rem;
+    border-bottom-right-radius: 0.25rem;
+    padding: 0rem 2.5rem 1.75rem 2.5rem;
+    background-color: white;
+    flex: 1 1 auto;
+    flex-direction: column;
+    display: flex;
+    width: 100%;
 }
 
-.theme-btn-provider:hover {
-  color: #999;
+.theme-btn-provider {
+    display: flex;
+    align-items: stretch;
+    background: white;
+    color: #0584c7;
+    letter-spacing: 0.046875rem;
+    min-width: 8rem;
+    padding: 0.375rem 1rem;
+    width: 100%;
+    font-size: 0.875rem;
+    font-weight: 400;
+    line-height: 1.5rem;
+    text-align: center;
+    justify-content: center;
+    vertical-align: middle;
+    cursor: pointer;
+    -webkit-user-select: none;
+    user-select: none;
+    border: 1px solid #0584c7;
+    border-radius: 0.25rem;
+    background-color: transparent;
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out,
+        border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+    box-shadow: none;
+}
+
+.theme-btn-provider:hover,
+.theme-btn-provider:focus {
+    background-color: #e8f8ff;
+}
+
+.theme-btn-provider:active {
+    background-color: #a6dcf9;
 }
 
 .theme-btn--primary {
-  background-color: #333;
-  border: none;
-  color: #fff;
-  min-width: 200px;
-  padding: 6px 12px;
+    background-color: #0584c7;
+    border: none;
+    color: #fff;
+    min-width: 200px;
+    padding: 0.5rem 1rem;
 }
 
 .theme-btn--primary:hover {
-  background-color: #666;
-  color: #fff;
+    background-color: #45a7d8;
+    color: #fff;
 }
 
 .theme-btn--success {
-  background-color: #2FC98E;
-  color: #fff;
-  width: 250px;
+    background-color: #28a197;
+    color: #fff;
+    width: 250px;
 }
 
 .theme-btn--success:hover {
-  background-color: #49E3A8;
+    background-color: #55c5bc;
 }
 
 .theme-form-row {
-  display: block;
-  margin: 20px auto;
+    display: flex;
+    flex-wrap: wrap;
+    flex-basis: 100%;
+    margin-bottom: 1rem;
+}
+
+.theme-form-row a {
+    display: block;
+    width: 100%;
+    text-decoration: none;
 }
 
 .theme-form-input {
-  border-radius: 4px;
-  border: 1px solid #CCC;
-  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
-  color: #666;
-  display: block;
-  font-size: 14px;
-  height: 36px;
-  line-height: 1.42857143;
-  margin: auto;
-  padding: 6px 12px;
-  width: 250px;
+    border-radius: 4px;
```

Comment:
It'd probably be easier to avoid conflicts and in general keep up with upstream if we use the same indentation and formatting as them @fuufen

Reply:
Yes, didn't think of that. Fixed it now.